auto-spacing.txt

```diff
@@ -413,16 +478,30 @@
 1FF2..1FF4     ; N  # N   GREEK SMALL LETTER OMEGA WITH VARIA AND YPOGEGRAMMENI..GREEK SMALL LETTER OMEGA WITH OXIA AND YPOGEGRAMMENI
 1FF6..1FFC     ; N  # N   GREEK SMALL LETTER OMEGA WITH PERISPOMENI..GREEK CAPITAL LETTER OMEGA WITH PROSGEGRAMMENI
 2010           ; O  # A   HYPHEN
-2013..2016     ; O  # A   EN DASH..DOUBLE VERTICAL LINE
+2013..2015     ; O  # A   EN DASH..HORIZONTAL BAR
+2016           ; C  # A   DOUBLE VERTICAL LINE
+2017           ; C  # N   DOUBLE LOW LINE
 2018..2019     ; O  # A   LEFT SINGLE QUOTATION MARK..RIGHT SINGLE QUOTATION MARK
 201C..201D     ; O  # A   LEFT DOUBLE QUOTATION MARK..RIGHT DOUBLE QUOTATION MARK
-2020..2022     ; O  # A   DAGGER..BULLET
-2024..2027     ; O  # A   ONE DOT LEADER..HYPHENATION POINT
-2030           ; O  # A   PER MILLE SIGN
-2032..2033     ; O  # A   PRIME..DOUBLE PRIME
-2035           ; O  # A   REVERSED PRIME
-203B           ; O  # A   REFERENCE MARK
-203E           ; O  # A   OVERLINE
+2020..2021     ; O  # A   DAGGER..DOUBLE DAGGER
+2022           ; C  # A   BULLET
+2023           ; C  # N   TRIANGULAR BULLET
+2024..2025     ; C  # A   ONE DOT LEADER..TWO DOT LEADER
+2026           ; O  # A   HORIZONTAL ELLIPSIS
+2027           ; C  # A   HYPHENATION POINT
+2030           ; C  # A   PER MILLE SIGN
+2031           ; C  # N   PER TEN THOUSAND SIGN
+2032..2033     ; C  # A   PRIME..DOUBLE PRIME
+2034           ; C  # N   TRIPLE PRIME
+2035           ; C  # A   REVERSED PRIME
+2036..2038     ; C  # N   REVERSED DOUBLE PRIME..CARET
+203B           ; C  # A   REFERENCE MARK
+203C..203D     ; C  # N   DOUBLE EXCLAMATION MARK..INTERROBANG
+203E           ; C  # A   OVERLINE
+2041..2043     ; C  # N   CARET INSERTION POINT..HYPHEN BULLET
+2047..2051     ; C  # N   DOUBLE QUESTION MARK..TWO ASTERISKS ALIGNED VERTICALLY
```

Comment:
@kidayasuo @xfq These are in cp932 (Shift-JIS), is `C` ok?
U+2047 DOUBLE QUESTION MARK
U+2048 QUESTION EXCLAMATION MARK
U+2049 EXCLAMATION QUESTION MARK